test/test_video_reader.py

```diff
@@ -1187,7 +1175,7 @@ def test_probe_video_from_memory(self):
 
     def test_probe_video_from_memory_script(self):
         scripted_fun = torch.jit.script(io._probe_video_from_memory)
-        self.assertIsNotNone(scripted_fun)
+        assert scripted_fun
```

Comment:
in order to be slightly more explicit about what we don't want scripted_fun to be, let's use `assert scripted_fun is not None`